Src/WitsmlExplorer.Api/Repositories/CosmosRepository.cs

```diff
@@ -35,7 +34,7 @@ public async Task<TDocument> GetDocumentAsync(TDocumentId id)
             var container = cosmosClient.GetContainer(dbName, containerId);
             var itemResponse = await container.ReadItemAsync<TDocument>(id.ToString(), new PartitionKey(id.ToString()));
```

Comment:
Might as well return the `itemResponse` directly without storing it as a variable